src/app/utils/utils.js

```diff
@@ -11,6 +11,9 @@ export function generateEntityIdentifier(
   baseAppURL,
   disableLink = false
 ) {
+  if (!namespace) {
+    return [];
```

Comment:
This should return `null`. An array is of a different type than a `JSX.Element` which is what this normally returns. If this was in TS it would have complained ;)

Reply:
Sure, `tableRows.js` does this a lot so thought it was the way you guys do it.

Reply:
Ahh np, so the difference there is that on the 'green' path it will return an array of data. So if there is missing data it'll still return an empty array (the same type of data).